hyperjumptech/monika
# Chore: extract notification to workspace package

# Monika Pull Request (PR)  

## What feature/issue does this PR add  
To make the code more modular, we need to extract notifications related code to separerate npm package so that it can be reused

## How did you implement / how did you fix it  
1.  Extract the notification channels code to monorepo workspace in folder `packages/notification`
2. update dependencies
3. update import path

## How to test  
1. run `npm install`
2. run `npm run build -w packages/notification`
3. run monika `npm start` with defined config
4. see notification is sent as is

## Notes
Many changes is just updating the import path. don't be overwhelmed :)
